_includes/manuals/nightly/1.2_release_notes.md

```diff
@@ -6,6 +6,10 @@ This section will be updated prior to the next release.
 
 ### Upgrade warnings
 
+#### New kind of facts in Discovery plugin
+
+Discovered hosts have a new kind of fact (discovery) so facts will no longer appear on the Monitor - Facts page. To search for discovery hosts, use the search box on the Hosts - Discovered page.
```

Comment:
Do you mean that it's a new (dedicated) fact origin?

Reply:
Yes, there a new kind of fact name (`DiscoveryFactName`) in Discovery.

Reply:
Then I'd use "fact origin" instead of "kind of facts". I think original also comes back in the UI and API so that's consistent. When I read "kind of facts" I thought it was an actual fact that provided more data.